### ISSUE TITLE
CLIP SEMI needs an update for Live 11

### ISSUE THREAD

**Issue by ldrolez** [ldrolez/clyphx-live11#5]
Live 11 displays a warning

**Comment by light2peter** [comment]
This also applies to everything in ClyphXClipActions.py right? e.g. any CLIP NOTES action

**Comment by light2peter** [comment]
From my Log.txt:

```
2023-12-22T16:14:57.729005: error: RemoteScriptError: <string>:2: DeprecationWarning: Live.Clip.Clip.get_selected_notes is deprecated since Live 11.0. Please use Live.Clip.Clip.get_selected_notes_extended instead.

2023-12-22T16:14:57.729121: error: RemoteScriptError: <string>:2: DeprecationWarning: Live.Clip.Clip.replace_selected_notes is deprecated since Live 11.0. Please use Live.Clip.Clip.apply_note_modifications instead.

```

**Comment by ldrolez** [comment]
closed by 43015fd69c49810a462dc5b85b9c074403899e5d
